op-e2e/setup.go

```diff
@@ -248,7 +248,7 @@ type EthInstance interface {
 }
 
 type System struct {
-	cfg SystemConfig
+	Cfg SystemConfig
```

Comment:
It seems worthwhile just to embed the `SystemConfig` struct here so we can avoid repeating the `.Cfg` field chaining.

Reply:
hmm, I don't think that makes sense. For example, we'd wind up with `System` having a `Start` method that instead of starting the system would create a new `System` and start that instead.  The system and the config are distinct entities and we shouldn't blur those responsibilities just to save a few characters.

Reply:
`System.Start` makes a lot of sense to me, but ok